main.cpp

```diff
@@ -89,15 +97,10 @@ struct Numeric
     See the previous hint for implementing the conversion functions for the Temporary if you want to get the held value
  */
 
-#include <iostream>
-#include <cmath>
-#include <functional>
-#include <memory>
-
-template<typename numType>
+template<typename NumericType>
 struct Numeric
 {
-    using myType = numType;
+    using myType = Temporary<NumericType>; // 
```

Comment:
- [ ] `using` directives follow the same rules as Type Names regarding capitalization.